Tests/Tests/Matchers/CDTQContainsAllElementsMatcher.m

```diff
@@ -31,6 +31,16 @@
         break;
     }
     
+    if (matches) {
```

Comment:
I think we can save this loop by doing this before the first pass of the arrays:

```
if([actual count] != [expected count]){
        return NO;
}
```